test/unit/app/browser/reducers/autoplayReducerTest.js

```diff
@@ -239,8 +258,33 @@ describe('autoplayReducer unit tests', function () {
     })
   })
 
+  describe('Always deny', function () {
+    before(function () {
+      translationSpy.reset()
+      showNotificationSpy.reset()
+      autoplayMedia = autoplayOption.ALWAYS_DENY
+      autoplayReducer(Immutable.fromJS({
+        siteSettings: {}
+      }), Immutable.fromJS({
+        actionType: appConstants.APP_AUTOPLAY_BLOCKED,
+        tabId: tabId
+      }))
+    })
+
+    it('no calls local.translation', function () {
+      assert(translationSpy.notCalled)
+      assert(translationSpy.notCalled)
+      assert(translationSpy.notCalled)
```

Comment:
why are we asserting three times here?

Reply:
oops, I must yanked and pasted too many times 😛